File: c/problem1.c
#include <stdio.h>


/*	https://projecteuler.net/problem=1	*/
int main()
{
	unsigned int i;
	unsigned int res = 0;

	for( i = 0 ; i < 1000; i++ )
	{
		if ( i % 3 == 0 || i % 5 == 0 )
		{
			res = res + i;
		}
	}

	// result
	printf("%d", res);


	return 0;
}

File: python/problem1.py
"""
Link:
    https://projecteuler.net/problem=1

Problem:
    If we list all the natural numbers below 10 that are multiples of 3 or 5,
    we get 3, 5, 6 and 9. The sum of these multiples is 23. Find the sum of
    all the multiples of 3 or 5 below 1000.
"""


if __name__ == "__main__":
    res = 0
    
    for n in range(1000):
        if n % 3 == 0 or n % 5 == 0:
            res = res + n
            
    # result
    print(res)